docs/Examples/Button.example.purs

```diff
@@ -27,7 +27,7 @@ docs =
 
       , [ h2_ "Disabled"
         , example
-            $ button defaults { title = "Button", disabled = true }
+            $ button defaults { title = "Button", buttonState = Just $ Disabled }
```

Comment:
No need for the `$` here

Reply:
Lol, yeah I know (I think I just do that out of habit). Will clean up though! Thx